gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2

```diff
@@ -1808,26 +1799,17 @@ def test_{{ method_name }}_rest_no_http_options():
 {% endmacro %}
 
 
-{% macro empty_call_test(method, method_name, service, api, uuid4_re, is_async=False) %}
+{% macro method_call_test(test_name, method, service, api, request, is_async=False) %}
```

Comment:
We should also make the name more specific, because most of our tests involve calling a method, but I take it this method will not be used in all our test cases?

Reply:
Tests in this code base have a lot of duplication. Ideally `method_call_test` will be used in the majority test cases, and where it can't be used, the name of other macros could be more descriptive. `method_call_test_<with_customization>`. 

I've updated `method_call_test` to `method_call_test_generic` to reflect that tests that can't use this could be named ``method_call_test_<with_customization>`.

Reply:
Addressed in https://github.com/googleapis/gapic-generator-python/pull/2126/commits/5204db8b051ae77b20470c8a88929f83fd373a52 and https://github.com/googleapis/gapic-generator-python/pull/2126/commits/374e444e61e9afd5e9f9b88524eecd89a827dcfe